### PR TITLE
SMP: Replace arrow down with upload icon for importing a site 

### PR DESCRIPTION
**Pull request by katinthehatsite**
Related to https://github.com/Automattic/wp-calypso/issues/84870

## Proposed Changes

This PR replaces the arrow down icon with the Gutenberg download icon for `Import an existing site` action on `/sites` page.

Screenshot 2023-12-21 at 11 13 31 AM

## Testing Instructions

* Navigate to Calypso Live Link
* Navigate to `/sites` page
* Click on the chevron by the `Add new site` button in the top right corner 
* Observe that dropdown with the `Impost an existing site` action
* Observe that the icon has been updated to `download` 
* Confirm that the icon is properly aligned

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/sites-dashboard/components/sites-dashboard.tsx [hunk 1]
```diff
@@ -219,8 +225,8 @@ export function SitesDashboard( {
 								recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
 							} }
 							href={ importSiteUrl }
-							icon="arrow-down"
 						>
+							<UploadIcon icon={ upload } size={ 18 } />
```
Comment (katinthehatsite) [inline client/sites-dashboard/components/sites-dashboard.tsx]: I set the size to be consistent with the Jetpack icon above but if it looks on a smaller side, we can increase to 20 which also looks good

Comment (danielbachhuber) [reply client/sites-dashboard/components/sites-dashboard.tsx]: This doesn't have the correct effect on hover:

CleanShot 2023-12-07 at 04 55 17@2x

Here's what I'd suggest instead:

```diff
diff --git a/client/sites-dashboard/components/sites-dashboard.tsx b/client/sites-dashboard/components/sites-dashboard.tsx
index f2159593868..440457d249f 100644
--- a/client/sites-dashboard/components/sites-dashboard.tsx
+++ b/client/sites-dashboard/components/sites-dashboard.tsx
@@ -145,11 +145,6 @@ const ManageAllDomainsButton = styled( Button )`
 	margin-inline-end: 1rem;
 `;
 
-const UploadIcon = styled( Icon )`
-	margin-right: 8px;
-	vertical-align: bottom;
-`;
-
 const SitesDashboardSitesList = createSitesListComponent();
 
 export function SitesDashboard( {
@@ -225,8 +220,8 @@ export function SitesDashboard( {
 								recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
 							} }
 							href={ importSiteUrl }
+							icon={ <Icon class="gridicon" icon={ upload } size={ 18 } /> }
 						>
-							<UploadIcon icon={ upload } size={ 18 } />
 							<span>{ __( 'Import an existing site' ) }</span>
 						</PopoverMenuItem>
 					</SplitButton>
```

**Review by danielbachhuber — CHANGES_REQUESTED** [review]

**Comment by wojtekn** [comment]
@danielbachhuber @katinthehatsite should we hold work on this one until we agree about the change under another ticket? We are making a change that is obviously wrong.

**Comment by danielbachhuber** [comment]
> should we hold work on this one until we agree about the change under another ticket? We are making a change that is obviously wrong.

@wojtekn I thought we agreed upon the up arrow because it's consistent with Gutenberg? https://github.com/Automattic/wp-calypso/pull/84873#issuecomment-1841094505

**Comment by wojtekn** [comment]
> @wojtekn I thought we agreed upon the up arrow because it's consistent with Gutenberg? https://github.com/Automattic/wp-calypso/pull/84873#issuecomment-1841094505

Gutenberg icons are great, I meant the down/up direction thing.

**Comment by danielbachhuber** [comment]
> Gutenberg icons are great, I meant the down/up direction thing.

@wojtekn I thought we agreed upon "up"?

File: client/sites-dashboard/components/sites-dashboard.tsx (hunk 1 above)
Comment (katinthehatsite) [reply client/sites-dashboard/components/sites-dashboard.tsx]: Hi, thanks for the feedback. I saw the icon from Gridicon but I thought we were moving towards the Gutenberg icons to be consistent. I am happy to add the styles to the Gutenberg icon to look good on hover. 

@javierarce what is your opinion? Should we use the Gutenberg icon instead or it is fine with Gridicon?

Comment (javierarce) [reply client/sites-dashboard/components/sites-dashboard.tsx]: We should use the Gutenberg one (we are trying to move away from Gridicons as much as possible)

**Review by danielbachhuber — APPROVED** [review]
Looks good:

CleanShot 2023-12-21 at 13 52 21@2x

**Pull request merged**